interpreter.py

```diff
@@ -158,10 +166,27 @@ def exec_operation(self, instruction):
             x, y = self._get_args(args)
             assert isinstance(x, Float) and isinstance(y, Float)
             return Float(x.value / y.value)
-        elif opcode == LLVMFSub:
+        elif opcode == LLVMSRem:
             x, y = self._get_args(args)
-            assert isinstance(x, Float) and isinstance(y, Float)
-            return Float(x.value - y.value)
+            assert isinstance(x, Integer) and isinstance(y, Integer)
```

Comment:
I'm pretty sure that RPython's type inference won't allow `_get_args` to return both `(Integer, Integer` and `(Float, Float)`.